src/main/java/org/apache/commons/mail/EmailUtils.java

```diff
@@ -295,10 +295,6 @@ static String encodeUrl(final String input) throws UnsupportedEncodingException
         for (final byte c : input.getBytes(US_ASCII))
         {
             int b = c;
-            if (b < 0)
```

Comment:
This is not dead code. It converts a signed 8-bit number (byte) to an unsigned 8-bit number, stored in an integer. It could be replaced with:
```Java
int b = c & 0xff;
```

Reply:
@aherbert done